hazelcast/src/main/java/com/hazelcast/scheduledexecutor/ScheduledTaskHandler.java

```diff
@@ -22,29 +22,27 @@
 import com.hazelcast.spi.annotation.Beta;
 
 /**
- * Resource handler pointing to a {@link IScheduledFuture}.
- * The handler is used to interact with the <code>ScheduledFuture</code> in a {@link IScheduledExecutorService}.
+ * Resource handler pointing to a {@link IScheduledFuture}. The handler is used to interact with the <code>ScheduledFuture</code>
+ * in a {@link IScheduledExecutorService}.
  *
- * To access the handler, see {@link IScheduledFuture#getHandler()}.
- * To re-acquire access to a previously scheduled task, having only the handler at hand,
- * see {@link IScheduledExecutorService#getScheduledFuture(ScheduledTaskHandler)}
+ * <p>To access the handler, see {@link IScheduledFuture#getHandler()}. To re-acquire access to a previously scheduled task,
```

Comment:
Maybe leave the `<p>` on a separate line

Reply:
I think this creates clutter for just the `p`. But I have no strong preference. Even JDK code has it in both places